src/coreclr/vm/ceeload.h

```diff
@@ -1428,8 +1428,6 @@ class Module
 
         BAD_FORMAT_NOTHROW_ASSERT(TypeFromToken(token) == mdtTypeDef);
 
```

Comment:
The ProfilingBlobEntry type and all its derivatives should go. IbcNameHandle should go. 
ProfilingBlobTable should go. TokenProfileData should go. 

m_nativeImageProfiling should go
m_methodProfileList should be deleted

CORCOMPILE_METHOD_PROFILE_LIST should go

Delete corbbtprof.h

CorProfileData should go.

Remove the HOT_MODEL from metadata

Reply:
We should just delete the entire ProfileDataNode.cs file, its tied into the /tuning switch to crossgen2.

Reply:
Done.